src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp

```diff
@@ -302,6 +302,39 @@ ov::Tensor create_and_fill_tensor_consistently(const ov::element::Type element_t
     return tensor;
 }
 
+template <typename T>
+void fill_data_sin_value(T* data, size_t size) {
```

Comment:
is it required? I saw such in only one test and added it inplace

Reply:
Oh. Yes, it is unused anymore

Reply:
removed